app/Models/Contest.php

```diff
@@ -61,11 +61,16 @@ public function votes()
         return $this->hasMany(ContestVote::class);
     }
 
-    public function isBestOf()
+    public function isBestOf(): bool
     {
         return isset($this->getExtraOptions()['best_of']);
     }
 
+    public function isSubmittedBeatmaps(): bool
+    {
+        return isset($this->getExtraOptions()['submitted_beatmaps']);
```

Comment:
hmm, bestof uses an isset check as well, think the mechanism is similar here, why would submitted_beatmaps be set to false when it can be just not set at all? :thinking:

Reply:
value of `best_of` is object containing some more metadata...